src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -1,66 +1,69 @@
 /* eslint complexity: [2, 25] */
-import { FORMS } from './RemediationConstants';
+import { AUTHENTICATOR_KEY, FORMS } from './RemediationConstants';
 
 const FORMNAME_CLASSNAME_MAPPINGS = {
   [FORMS.IDENTIFY]: {
     [FORMS.IDENTIFY]: 'primary-auth',
-    'password': 'primary-auth'
+    [AUTHENTICATOR_KEY.PASSWORD]: 'primary-auth'
   },
   [FORMS.ENROLL_PROFILE]: {
     [FORMS.ENROLL_PROFILE]: 'registration',
   },
   [FORMS.CHALLENGE_AUTHENTICATOR]: {
-    email: 'mfa-verify-passcode',
-    password: 'mfa-verify-password',
+    [AUTHENTICATOR_KEY.EMAIL]: 'mfa-verify-passcode',
+    [AUTHENTICATOR_KEY.PASSWORD]: 'mfa-verify-password',
     sms: 'mfa-verify-passcode',
     voice: 'mfa-verify-passcode',
-    'security_question': 'mfa-verify-question',
-    'security_key': 'mfa-verify-webauthn',
-    app: 'mfa-verify',
+    [AUTHENTICATOR_KEY.SECURITY_QUESTION]: 'mfa-verify-question',
```

Comment:
I guess we need to add an entry for AUTHENTICATOR_KEY.RECOVERY_QUESTION to  'mfa-verify-question', as well.

Reply:
We only have `SECURITY_QUESTION` now